.github/workflows/main_workflow.yml

```diff
@@ -414,4 +414,4 @@ jobs:
 
       - name: Unit tests
         run: |
-          tox --develop -e packages-py${{ matrix.python-version }}-linux -- -m 'e2e'
+          tox --develop -e e2e-py${{ matrix.python-version }}-linux -- -m 'e2e'
```

Comment:
`packages-py3.10-linux` env doesn't run for agents, only for skills and connections, because of coverage collection. This is another env with the same setup as we had it before #1676